.github/workflows/nodejs.yml

```diff
@@ -11,4 +11,5 @@ jobs:
     # Documentation: https://github.com/zakodium/workflows#nodejs-ci
     uses: zakodium/workflows/.github/workflows/nodejs.yml@nodejs-v1
     with:
+      lint-check-types: true
       node-version-matrix: '[12, 14, 16]'
```

Comment:
Please remove this line. We now inherit the default version matrix in most projects